<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotePad</class>
 <widget class="QMainWindow" name="NotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/img/resources/_app_icon.ico</normaloff>:/img/resources/_app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionExport_to_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuChange">
    <property name="title">
     <string>Change</string>
    </property>
    <addaction name="actionFont_Color"/>
    <addaction name="actionBackground_Color"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionInsert_Date"/>
    <addaction name="actionInsert_Time"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuChange"/>
   <addaction name="menuInsert"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_Left"/>
   <addaction name="actionAlign_Center"/>
   <addaction name="actionAlign_Right"/>
   <addaction name="actionAlign_Justify"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionFont_Color"/>
   <addaction name="actionBackground_Color"/>
   <addaction name="separator"/>
   <addaction name="actionExport_to_PDF"/>
   <addaction name="actionPrint_Preview"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionInsert_Date"/>
   <addaction name="actionInsert_Time"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/new.png</normaloff>:/img/resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/open.png</normaloff>:/img/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/save.png</normaloff>:/img/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save this file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/save_as.png</normaloff>:/img/resources/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Cut.png</normaloff>:/img/resources/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Copy.png</normaloff>:/img/resources/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Paste.png</normaloff>:/img/resources/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlign_Left">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/text_align_left.png</normaloff>:/img/resources/text_align_left.png</iconset>
   </property>
   <property name="text">
    <string>Align Left</string>
   </property>
   <property name="toolTip">
    <string>Align text to the left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlign_Right">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/text_align_right.png</normaloff>:/img/resources/text_align_right.png</iconset>
   </property>
   <property name="text">
    <string>Align Right</string>
   </property>
   <property name="toolTip">
    <string>Align text to the right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlign_Center">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/text_align_center.png</normaloff>:/img/resources/text_align_center.png</iconset>
   </property>
   <property name="text">
    <string>Align Center</string>
   </property>
   <property name="toolTip">
    <string>Align text to the center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAlign_Justify">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/text_align_justify.png</normaloff>:/img/resources/text_align_justify.png</iconset>
   </property>
   <property name="text">
    <string>Align Justify</string>
   </property>
   <property name="toolTip">
    <string>Justify text alignment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/edit-bold.png</normaloff>:/img/resources/edit-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Make text bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/text_italic.png</normaloff>:/img/resources/text_italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italicize text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/underline_icon.png</normaloff>:/img/resources/underline_icon.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="toolTip">
    <string>Underline text</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Undo.png</normaloff>:/img/resources/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo recent activity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Redo.png</normaloff>:/img/resources/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo recent activity</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/font.png</normaloff>:/img/resources/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Change font options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFont_Color">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/change_text_color.png</normaloff>:/img/resources/change_text_color.png</iconset>
   </property>
   <property name="text">
    <string>Font Color</string>
   </property>
   <property name="toolTip">
    <string>Change font color</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/bg_color.png</normaloff>:/img/resources/bg_color.png</iconset>
   </property>
   <property name="text">
    <string>Background Color</string>
   </property>
   <property name="toolTip">
    <string>Change background color</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionInsert_Date">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Calendar.png</normaloff>:/img/resources/Calendar.png</iconset>
   </property>
   <property name="text">
    <string>Insert Date</string>
   </property>
   <property name="toolTip">
    <string>Insert today's date</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionInsert_Time">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Clock.png</normaloff>:/img/resources/Clock.png</iconset>
   </property>
   <property name="text">
    <string>Insert Time</string>
   </property>
   <property name="toolTip">
    <string>Insert current time</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExport_to_PDF">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/doc_pdf.png</normaloff>:/img/resources/doc_pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export to PDF</string>
   </property>
   <property name="toolTip">
    <string>Export document as PDF</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Preview.png</normaloff>:/img/resources/Preview.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
   <property name="toolTip">
    <string>Open print preview</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/Print.png</normaloff>:/img/resources/Print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/select_all.png</normaloff>:/img/resources/select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select all text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/clear_all.ico</normaloff>:/img/resources/clear_all.ico</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="toolTip">
    <string>Clear all text</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/app_info.png</normaloff>:/img/resources/app_info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About app</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/img/resources/exit.png</normaloff>:/img/resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Quit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
